packages/google_maps_flutter/google_maps_flutter_web/lib/src/shims/dart_ui.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// This file shims dart:ui in web-only scenarios, getting rid of the need to
+/// suppress analyzer warnings.
+
+export 'dart_ui_fake.dart' if (dart.library.html) 'dart_ui_real.dart';
```

Comment:
Do we want a TODO to remove this with a link to https://github.com/flutter/flutter/issues/55000 ?

Reply:
Yes, I'll add a TODO to 55000, this change doesn't fix the underlying issue yet.